QUIRKS.md

```diff
@@ -0,0 +1,67 @@
+# FFmpeg Quirks in LPMS
+This document outlines how LPMS tweaks FFmpeg transcoding pipeline to address some specific problems.
+
+## Handle zero frame (audio-only) segments at the start of a session
+
+### Problem
+Livepeer used to reject transcoding requests when it is sent audio-only segments. This could happen variety of reasons. In the case of MistServer, it happens when a transcode is enabled and there is more audio data in the buffer than video data. It could also happen in a variety of edge cases where bandwidth is severely constrained and end-user client software decides to only send audio to keep something online. 
+
+### Desired Behavior
+Instead of erroring on such segments, let's accept them and send back audio-only tracks. The audio output should be exactly the same as the audio input.
+
+### Our Solution
+We bypass the usual transcoding process for audio-only segments and just return them back. To do this, we check if video frame is present, and if not present just copy the frames to the output as it is. The bypassing is done by forcing the copy transcoder https://github.com/livepeer/lpms/commit/8bc28e3f702049a17c24ab2041857a47d8af51bf for such segments.

```

Comment:
While the by-pass check code is still implemented in LPMS we no longer use this function on the transcoder directly, or set the 'copy' encoder. https://github.com/livepeer/lpms/blob/46f7030c62d91d5f74b02bfa13dca42eb8ad8fa5/ffmpeg/extras.c#L110-L117 

We instead moved the calling logic on the Broadcaster side in https://github.com/livepeer/go-livepeer/pull/1933 and the document should reflect that.